.eslintrc.js

```diff
@@ -37,18 +38,18 @@ module.exports = {
 	// check if imports actually resolve
 	settings: {
 		'import/resolver': {
-			'webpack': {
-				'config': 'build/webpack.base.conf.js'
+			webpack: {
+				config: './build/webpack.base.conf.js'
```

Comment:
adding `./` to this path fixed the `Casing of graphql-tag does not match the underlying filesystem.eslintimport/no-unresolved` error in vscode